downstream/modules/platform/con-aap-upgrade-prereq.adoc

```diff
@@ -0,0 +1,147 @@
+[id="con-aap-upgrade-prereq_{context}"]
+
+= Prerequisites for upgrading {PlatformNameShort}
+
+== Node requirements
+
+The following specifications are required for the nodes involved in the {PlatformNameShort} upgrade process:
+
+* 16 GB of RAM for controller nodes, database node, execution nodes and hop nodes.
+* 4 CPUs for controller nodes, database nodes, execution nodes, and hop nodes.
+* 150 GB+ disk space for database node.
+* 40 GB+ disk space for non-database nodes.
+* DHCP reservations use infinite leases to deploy the cluster with static IP addresses.
+* DNS records for all nodes.
+* Red Hat Enterprise Linux 8 or later 64-bit (x86) installed for all nodes.
+* Chrony configured for all nodes.
+
+== {ControllerNameStart} configuration requirements
+
+.Configuring NTP server using Chrony
+
+Each {PlatformNameShort} node in the cluster must have access to an NTP server. Use the `chronyd` to synchronize the system clock with NTP servers. This ensures that when cluster nodes use SSL certificates that require validation, they do not fail if the date and time between the nodes are not in sync.
```

Comment:
Suggest rewording:
This ensures that cluster nodes using SSL certificates that require validation do not fail if the date and time between nodes are not in sync.